.github/workflows/hashlint.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: Python lint workflow
```

Comment:
Can you make this something like "Hash lint workflow" so it shows differently on the actions dashboard?